Reference/Searching/Examine/Indexing/index-v8.md

```diff
@@ -0,0 +1,352 @@
+---
+versionFrom: 8.0.0
+versionTo: 8.0.0
+---
+
+# Custom indexing
+
+:::note
+This document has been verified for Umbraco 8.
+
+If you are using Umbraco 9 or later versions, please refer to the note on the [Examine documentation landing page](../index.md) for more details.
+:::
+
+Examine has changed quite a bit in Umbraco 8 (and by "a bit" we really mean a lot). In Umbraco 7 everything was configured in the two Examine config files - in Umbraco 8 everything happens through C#.
+
+## Customizing the built in indexes
+
+You can modify the built in indexes in several ways:
+
+* Events - giving you control over exactly what data goes into them and how the fields are configured
+* Changing the field value types to change how values are stored in the index
+* Changing the `IValueSetValidator` to change what goes into the index
+* Take control of the entire index creation pipeline to completely change the implementation
+
+### Changing field value types
+
+By default Examine will store values into the Lucene index as "Full Text", meaning it will be indexed and analyzed for a textual search. However, if a value that you are storing in a field is numerical, a date/time or another specific value type then you might want to change how this is stored in the index. This way you will be able to take advantage of some features such as numerical range or date features, etc...
```

Comment:
```suggestion
By default, Examine will store values into the Lucene index as "Full Text", meaning it will be indexed and analyzed for a textual search. However, if a value that you are storing in a field is numerical, a date/time, or another specific value type then you might want to change how this is stored in the index. This way you will be able to take advantage of some features such as numerical range or date features, etc...
```